### PR TITLE
Write Acceptance tests

### LINKED ISSUE #38: Create tests for repository view
See #37

### PR DESCRIPTION
**Pull request by phyrog**
This will close #38, #39 and #41.

- [x] Integrate generated ember models
- [x] Redo mirage config
- [x] Write tests
  - [x] Index page
  - [x] Repository page
  - [x] User page
  - [x] Organization page
  - [x] New repository page
  - [x] Search page
- [x] Obey formatting rules

### CODE REVIEW

File: tests/acceptance/search-test.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+    it('resets the tabs correctly upon re-entering page', () => {
+      click('a:contains(Users)');
+      click('a:contains(Ontohub)');
+      click('a:contains(Search)');
+      andThen(() => {
+        expect(currentURL()).to.equal('/search');
+      });
+    });
+  });
```
Comment (eugenk) [inline tests/acceptance/search-test.js]: You could add a test that there are zero OMS. It should have a comment that this is displayed, but there is no further implementation.

File: tests/acceptance/user-test.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    it('resets the tabs correctly upon re-entering page', function() {
+      const organizationsTab = find(
+        '.top-route-header .tabs a:contains(Organizations)'
+      );
+      click(organizationsTab);
+      click(`.page-content a:contains(${this.test.organizations[0].name})`);
+      click('a:contains(Members)');
+      click(`.page-content a:contains(${this.test.user.name})`);
+      andThen(() => {
+        expect(currentURL()).to.not.equal(
```
Comment (eugenk) [inline tests/acceptance/user-test.js]: This one is redundant. It follows from the test in line 99.

File: tests/unit/models/repository-test.js [hunk 3]
```diff
@@ -6,19 +6,53 @@ import Ember from 'ember';
...
+  describe('belongs to', () => {
+    it('a user', function() {
+      let model = this.store().modelFor('repository'),
+          relationship = Ember.get(model, 'relationshipsByName').
+        get('ownerUser');
+      expect(relationship.key).to.equal('ownerUser');
+      expect(relationship.kind).to.equal('belongsTo');
+      Ember.run(() => {
+        let user = this.store().createRecord('user', { id: 'somebody' }),
+            subj = this.subject();
```
Comment (eugenk) [inline tests/unit/models/repository-test.js]: Why not use the full word `subject`?

File: tests/unit/models/user-test.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { setupModelTest } from 'ember-mocha';
+import Ember from 'ember';
+
+describe('Unit | Model | user', () => {
+  setupModelTest('user', {
+    // Specify the other units that are required for this test.
```
Comment (eugenk) [inline tests/unit/models/user-test.js]: Please remove this comment.

File: tests/unit/models/repository-test.js [hunk 5]
```diff
@@ -6,19 +6,53 @@ import Ember from 'ember';
 describe('Unit | Model | repository', () => {
   setupModelTest('repository', {
     // Specify the other units that are required for this test.
```
Comment (eugenk) [inline tests/unit/models/repository-test.js]: Please remove this comment.

File: tests/unit/models/organization-test.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { setupModelTest } from 'ember-mocha';
+import Ember from 'ember';
+
+describe('Unit | Model | organization', () => {
+  setupModelTest('organization', {
+    // Specify the other units that are required for this test.
```
Comment (eugenk) [inline tests/unit/models/organization-test.js]: Please remove this comment.

File: tests/acceptance/organization-test.js [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+    destroyApp(application);
+  });
+
+  describe('Organization page', () => {
+    beforeEach(function() {
+      const memberCount = 3,
+            repositoryCount = 2,
+            members = server.createList('user', memberCount),
+            organization = server.create('organization', {
+              memberIds: _.map(members, (u) => u.id)
```
Comment (eugenk) [inline tests/acceptance/organization-test.js]: According to http://www.ember-cli-mirage.com/docs/v0.3.x/models/#associations it should work (if the factories call the same methods as described there).

File: .travis.yml [hunk 8]
```diff
@@ -10,17 +10,18 @@ cache:
     - "/home/travis/.yarn-cache"
 
 before_install:
-  - "npm config set spin false"
-  - "npm --version"
+  - "yarn --version"
   - "phantomjs --version"
+  - "git clone https://github.com/ontohub/ontohub-backend ../ontohub-backend"
+  - "ln -s $(readlink -f ../ontohub-backend/spec/support/api/schemas) app/schemas"
```
Comment (phyrog) [inline .travis.yml]: I'll create an issue after this PR is merged to only link the `models` and `relationships` directories during delopyment.

File: app/adapters/repository.js [hunk 9]
```diff
@@ -13,6 +13,10 @@ export default ApplicationAdapter.extend({
   _urlForRecord(id) {
     let host = this.get('host'),
         url = `/repositories/${id}`;
+    /*
+     * This is configuration specific and can't be tested
+     */
+    /* istanbul ignore else */
     if(host) {
       url = `${host}${url}`;
```
Comment (phyrog) [inline app/adapters/repository.js]: I opened #76 for this.

File: tests/acceptance/index-test.js [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+  });
+
+  it('shows the welcome message', () => {
+    const header = find('.top-route-header'),
+          heading = find('h1', header),
+          subheading = find('p.help-text', header);
+
+    expect(heading.text()).to.equal('Welcome to Ontohub!');
+    // eslint-disable-next-line max-len
+    expect(subheading.text()).to.equal('Repositories and proof tools for Ontologies, Models and Specifications (OMS)');
```
Comment (phyrog) [inline tests/acceptance/index-test.js]: We agreed to use multiple `expect`s per `it` (for ease of use) but keep `expect`s pure.

File: tests/acceptance/search-test.js (hunk 1 above)
Comment (phyrog) [inline tests/acceptance/search-test.js]: Done in f857206

File: tests/acceptance/repository-test.js [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+      click(description);
+      fillIn('.top-route-header form input', newDescription);
+      click('.top-route-header form button');
+      andThen(() => {
+        expect(find('span', header).text()).to.equal(newDescription);
+      });
+    });
+  });
+
+  describe('New repository page', () => {
```
Comment (ebolloff) [inline tests/acceptance/repository-test.js]: Please use lowercase in all steps
